### DISCUSSION TITLE
Error on next.config.js

### DISCUSSION THREAD

**Discussion by hoi82** [vercel/next.js#45894] [Help]
I got an error message, 

Parsing error: [BABEL] /Users/.../next.config.js: require(...).pluginSyntaxImportAssertions is not a function (While processing: "/Users/.../node_modules/next/babel.js")eslint 

Here is codes of next.config.js,

const withBundleAnalyzer = require('@next/bundle-analyzer')({
    enabled: process.env.ANALYZE === 'true'
});

/** @type {import('next').NextConfig} */
module.exports = withBundleAnalyzer({
    poweredByHeader: false,
    i18n: {
        locales: ["en", "ko", "ja"],
        defaultLocale: "en"
    }
});

How can I fix it? I'm using version 13.1.6

**Comment by ciruz** [comment]
Hey,

next.js includes a next/babel preset, try this config:

```javascript
const withBundleAnalyzer = require('@next/bundle-analyzer')({
  enabled: process.env.ANALYZE === 'true',
})
module.exports = withBundleAnalyzer({
  poweredByHeader: false,
  i18n: {
    locales: ["en", "ko", "ja"],
    defaultLocale: "en"
  },
  babel: {
    presets: ['next/babel']
  }
});
```
